### ISSUE TITLE
Windows file formats UTF-8-BOM (CR LF)

### ISSUE THREAD

**Issue by cs-NET** [leolabs/json-autotranslate#14]
Ran into an issue running json-autotranslate for the second time on Windows platform. It appears that when reading the files from the cache folder an exception will be thrown parsing JSON if the files are not UTF-8 with Unix (LF) EOL. In my case one of the files was UTF-8-BOM with Windows (CR LF) EOL, I assume that the file format is carried over from the original file. (I happen to auto generate one of the json files from a RESX file hence one file was in a windows file format).

To fix the problem I manually converted the JSON files to UTF-8 with Unix (LF) EOL, including those in the cache folder.

Adding a note about Windows compatibility would be sufficient, or even better adding support for windows file formats. 

Thanks for the great work!

Version 1.5.2
```
Unexpected token ﻿ in JSON at position 0
SyntaxError: Unexpected token ﻿ in JSON at position 0
    at JSON.parse (<anonymous>)
    at D:\repos\project\node_modules\json-autotranslate\lib\index.js:248:91
```

**Comment by leolabs** [comment]
Hey, sorry for the late reply. I'll try to fix that issue today. Could you attach a sample JSON file where this error occurs so that I can reproduce the issue on my end? :)

**Comment by cs-NET** [comment]
I have attached a json file in utf-8-bom with cr lf 

[windows-utf-8-bom-cr-lf.zip](https://github.com/leolabs/json-autotranslate/files/3897662/windows-utf-8-bom-cr-lf.zip)

**Comment by leolabs** [comment]
This should be fixed in c45a44b :)
